lab4-new.java/src/StringSearch.java

```diff
@@ -31,15 +31,21 @@ public static ArrayList<String> search(String word, File file) {
     }
 
 
-    public static String checkLine(String word, String line) {
-        if(line.contains(word)) {
+    public static String checkLine(String word, String line, Boolean ignoreCase) {
+        if(ignoreCase && line.toLowerCase().contains(word.toLowerCase())) {
+            return line;
+        } else if(line.contains(word)) {
             return line;
         }
         return null;
     }
 
     public static void main(String[] args) {
-        System.out.println(search(args[0], new File(args[1])));
+        Boolean ignoreCase = false;
+        if(args.length == 3 && args[2].equals("-i")) {
```

Comment:
Provide supportive feedback to user when there is a wrong input.

Reply:
Fixed